### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by saraedum** [nthiery/recursive-monkey-patch#4]
Running `python setup.py test`, I get
```
160         ['A', 'B', '__builtins__', ...]
161 
162     .. RUBRIC:: Automatizing the monkey patching
163 
164     A typical idiom is to put the :func:`monkey_patch` call in the
165     patching module initialization (see the code of
166     :mod:`a_test_module2.__init__` for an example). Then the user only
167     has to import the patching module::
168 
169         >>> import a_test_module_patch2
UNEXPECTED EXCEPTION: ModuleNotFoundError("No module named 'a_test_module_patch2'",)
Traceback (most recent call last):
```

**Comment by nthiery** [comment]
I committed an alternative fix: 008ce1e5bf .

In fact that was my original implementation which was in files I had not committed and just rediscovered ... blame on me ...

**Comment by nthiery** [comment]
Waiting for feedback from  CI, and then I'll go on to prepare on a release.
